doc/man1/cms.pod

```diff
@@ -553,6 +559,39 @@ The B<-debug_decrypt> option can be used to disable the MMA attack protection
 and return an error if no recipient can be found: this option should be used
 with caution. For a fuller description see L<CMS_decrypt(3)>).
 
+=head1 CAdES Basic Electronic Signature (CAdES-BES)
+
+A CAdES Basic Electronic Signature (CAdES-BES) contains:
+
+=over 4
+
+=item *
+
+The signed user data as defined in CMS (RFC 3852);
+
+=item *
+
+Content-type of the EncapsulatedContentInfo value being signed;
+
+=item *
+
+Message-digest of the eContent OCTET STRING within encapContentInfo being signed;
+
+=item *
+
+ESS signing-certificate or ESS signing-certificate-v2 defined in Enhanced Security Services (ESS), RFC 2634 and RFC 5035. 
```

Comment:
_An_ ESS signing-certificate or ESS signing-certificate-v2 _attribute, as_ defined in ...